src/locales/en-US/list.ftl

```diff
@@ -191,6 +191,8 @@ modal-delete = Delete this login?
   .cancelLabel = Cancel
 
 connect-another-device = Connect another device
```

Comment:
We need a new ID for this string when adding attributes. The string is translated in Pontoon and FTL files, nobody will know it changed if we don't change the ID.